terraform/files/kubernetes-manifests.d/harbor/base/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- https://github.com/SovereignCloudStack/k8s-harbor//base/?ref=v5.1.0
```

Just noticed this. Should there be a double `//`?
```suggestion
- https://github.com/SovereignCloudStack/k8s-harbor/base/?ref=v5.1.0
```

---

It can be seen also [here](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/resource/) - click the *[hashicorp URL](https://github.com/hashicorp/go-getter#url-format) format*

---

Ad your provided [guide](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#remote-directories)
> `//` to delimits the root of the repo.

Interesting.